spec/unit/mutant/integration/rspec_spec.rb

```diff
@@ -115,10 +115,20 @@
   end
 
   before do
-    expect(RSpec::Core::ConfigurationOptions).to receive(:new).with(%w[spec --fail-fast]).and_return(options)
-    expect(RSpec::Core::Runner).to receive(:new).with(options).and_return(runner)
-    expect(RSpec).to receive(:world).with(no_args).and_return(world)
-    allow(Time).to receive(:now).and_return(Time.now)
+    expect(RSpec::Core::ConfigurationOptions).to receive(:new)
+      .with(%w[spec --fail-fast])
+      .and_return(rspec_options)
+
+    expect(RSpec::Core::Runner).to receive(:new)
+      .with(rspec_options)
+      .and_return(rspec_runner)
+
+    expect(RSpec).to receive(:world)
+      .with(no_args)
+      .and_return(world)
+
+    allow(Time).to receive(:now)
```

Comment:
To further express my point, if we begin to asset a method does not receive arguments when it already cannot, then it's not really different than asserting that a method doesn't receive a block. Technically, I guess you could assert these things, but I think it's redundant.